### DISCUSSION TITLE
output error

### DISCUSSION THREAD

**Discussion by XueZhengHe03** [ExtremeFLOW/neko#1190] [Q&A] [answered]
I ./neko tgv.case , and the case nsamples = 0, it doesnt have the output  such as field0.f00000, field0.f00001 and field0.nek5000
but when I change the nsamples = 1,  it outputs something normally ：field0.f00000, field0.f00001 and field0.nek5000
Would you like to explain to me why? I would be grateful ！

**Comment by timofeymukha** [comment]
I guess this is expected behaviour? I.e. nsamples = 0 means never sample and output the fields?

**Comment by XueZhengHe03** [comment]
> 我想这是预期的行为吗？即 nsamples = 0 表示从不采样和输出字段？

maybe，could you tell me what is the meaning of the nsamples in the casefile

**Comment by timofeymukha** [comment, marked as answer]
Number of 3d field outputs during the simulation. There is one extra, corresponding to initial conditions.
